ckanext/dcat/tests/test_json_harvester.py

```diff
@@ -238,3 +270,55 @@ def test_not_copied_because_completely_different(self):
             harvested_dataset,
         )
         eq_(harvested_dataset['resources'][0].get('id'), None)
+
+
+class TestImportStage:
+
+    @classmethod
+    def setup_class(cls):
+        h.reset_db()
+
+    class MockHarvestObject:
+        guid = 'test_guid'
+        content = TestDCATJSONHarvestFunctional.json_content_invalid_tags
```

Comment:
Maybe create a constructor and pass the content when creating the object. For now we only have this one test that makes use of this, but it might be handy for other tests as well. WDYT?

Reply:
It's possible but I also think that it's not worth over engineering things until you actually need them, so if another test comes along that also wants to use this MockHarvestObject and to pass in different content then its worth making it more generic.

Reply:
I fully agree